common/src/core/java/xyz/wagyourtail/jsmacros/core/library/impl/FJsMacros.java

```diff
@@ -316,8 +316,8 @@ protected void openUrl(URL url) throws IOException {
      * @see IEventListener
      * 
      * @since 1.2.7
-     * @param event
-     * @param callback calls your method as a {@link java.util.function.BiConsumer BiConsumer}&lt;{@link BaseEvent}, {@link EventContainer}&gt;
+     * @param event #Event#
```

Comment:
should these #things# show up in the webdoc, we probably want to modify the webdoc so they either go away or get renamed or something

Reply:
not a problem anymore. switched to DocletReplace